sprite.cpp

```diff
@@ -42,18 +42,21 @@ void Sprite::draw(Context &ctx) {
 }
 
 void Sprite::update(Context &ctx) {
-	auto wrap = [](float n, float min, float max) -> float {
-		if (n < min) {
-			return max - (min - n);
-		} else if (n > max) {
-			return min + (n - max);
+	auto wrap = [](float home, float total, float min, float max) -> float {
```

Comment:
I'm not gonna hold up the PR over this but note to self, on some cleanup pass we should maybe change the name of this function and its parameters because the true intent is really confusing here.